cognite_toolkit/_cdf_tk/commands/dump_resource.py

```diff
@@ -90,54 +90,56 @@ def __init__(self, client: ToolkitClient, identifier: DataModelId | None = None)
         self.space_ids: set[str] = set()
 
     def _interactive_select(self) -> DataModelId:
-        include_global = False
-        spaces = self.client.data_modeling.spaces.list(limit=-1, include_global=include_global)
-        selected_space: str = questionary.select(
-            "In which space is your data model located?", [space.space for space in spaces]
-        ).ask()
-
```

Comment:
This listing of spaces include spaces without a data model (typically instances spaces), thus I changed it to only display the spaces that contain a data model.